tests/compile-fail/unaligned_pointers/unaligned_ptr3.rs

```diff
@@ -2,7 +2,8 @@
 // compile-flags: -Zmiri-disable-validation -Zmiri-disable-stacked-borrows
 
 fn main() {
-    for _ in 0..10 { // Try many times as this might work by chance.
+    for _ in 0..10 {
+        // Try many times as this might work by chance.
```

Comment:
Another case of rusfmt moving a comment to the wrong spot (the next few tests in the diff have the same issue)